src/Windowing/Silk.NET.Windowing.Common/Window.cs

```diff
@@ -19,12 +19,25 @@ public static class Window
         private const string SdlBackendNamespace = "Silk.NET.Windowing.Sdl";
         private const string GlfwBackendName = "GlfwPlatform";
         private const string SdlBackendName = "SdlPlatform";
+        private const string FallbackWindowClass = "SilkNET_App";
```

Comment:
Why `_App`? Why can't it just be `Silk.NET` or `silkdotnet`? Is there some naming convention I'm missing?

Reply:
I haven't found much about naming conventions or restrictions. Only control characters are not allowed. SDL uses "SDL_App". But we can just use a name without it. I think the '.' is ok but I generally avoid all characters beside letters, digits and underscores when I deal with names that are processed by different tools and systems. This makes it more robust imo. Can you change the name to something you prefer please?

Reply:
Looking at the documentation it seems like a dot should be allowed, so let's go with that I think. If it proves problematic we can change later.

```suggestion
        private const string FallbackWindowClass = "Silk.NET";
```